osu.Framework.iOS/Input/IOSTextInput.cs

```diff
@@ -59,16 +53,13 @@ public void EnsureActivated()
             // TODO: add iOS implementation
         }
 
-        public event Action<string> OnNewImeComposition
+        public void Deactivate()
         {
-            add { }
-            remove { }
+            view.KeyboardTextField.HandleShouldChangeCharacters -= handleShouldChangeCharacters;
+            view.KeyboardTextField.UpdateFirstResponder(false);
         }
 
-        public event Action<string> OnNewImeResult
-        {
-            add { }
-            remove { }
-        }
+        public event Action<string> OnNewImeComposition { add { } remove { } }
+        public event Action<string> OnNewImeResult { add { } remove { } }
```

Comment:
Nitpick / personal opinion but I much preferred the old formatting (goes for all places that have these one-lined).

Reply:
These can be replaced with raw `event`s:

```suggestion
        public event Action<string> OnNewImeComposition;
        public event Action<string> OnNewImeResult;
```

Reply:
Since this has been brought up twice I've applied the above suggestion in the relevant places.

Reply:
Never mind, this trips inspections due to unused events. I even ran inspectcode locally but it looked like they weren't going to be reported...